types/span.go

```diff
@@ -14,7 +13,7 @@ import (
 //   values, respectively.
 type Span struct {
 	CoreSpanMetadata
-	Annotations       []*Annotation          `json:"annotations,omitempty"` // TODO lift annotation struct definition into this file
+	Annotations       []*SpanAnnotation      `json:"annotations,omitempty"` // TODO lift annotation struct definition into this file
```

Comment:
Added a Span prefix to separate the interface/data structures used by Honeycomb from the structures used by Zipkin